### ISSUE TITLE
Turning phone from portrait to landscape

### ISSUE THREAD

**Issue by patlx** [aj3423/SpamBlocker#109]
Hi, 

Thank you so much for your app. I've got this issue while testing an SMS regexp. The testing window was open and to have a better view of the text to test I've turn my phone from portrait to landscape.

```text
version: 1.18
java.lang.RuntimeException: Unable to start activity ComponentInfo{spam.blocker/spam.blocker.ui.main.MainActivity}: K1.h: Unable to instantiate fragment i2.y: could not find Fragment constructor
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4169)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4325)
	at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:6484)
	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:6341)
	at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:71)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2574)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8762)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
Caused by: K1.h: Unable to instantiate fragment i2.y: could not find Fragment constructor
	at e0.H.a(Unknown Source:58)
	at e0.T.a(Unknown Source:2)
	at e0.U.<init>(SourceFile:13)
	at e0.N.U(Unknown Source:252)
	at e0.N.b(Unknown Source:217)
	at b.f.a(Unknown Source:18)
	at b.n.onCreate(Unknown Source:30)
	at h.i.onCreate(Unknown Source:0)
	at spam.blocker.ui.main.MainActivity.onCreate(Unknown Source:6)
	at android.app.Activity.performCreate(Activity.java:8591)
	at android.app.Activity.performCreate(Activity.java:8570)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1384)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4150)
	... 15 more
Caused by: java.lang.NoSuchMethodException: i2.y.<init> []
	at java.lang.Class.getConstructor0(Class.java:2363)
	at java.lang.Class.getConstructor(Class.java:1759)
	at e0.H.a(Unknown Source:19)
	... 27 more
```

**Comment by aj3423** [comment]
The crash has been fixed in v1.19, but the text width doesn't expand in landscape mode, I'll fix that.

**Comment by aj3423** [comment]
fixed in v2.0
